.travis.yml

```diff
@@ -12,6 +12,8 @@ notifications:
   email: false
 before_install:
 - .travis/before_install.sh
+- curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.7.0
```

Comment:
It may be worth adding a bit more docuementation to the root README at this point...

1. yarn version or yarn upgrade commands... i.e. "PF React uses Yarn 1.7.0. You can upgrade with `brew upgrade yarn` (Mac) or `npm install -g yarn`". If you are using older yarn versions you will get an error:
```
error An unexpected error occurred: "patterns.map is not a function".
```
Another option might just be warning the user if using an older version of yarn?

2. A link to the other package README's in the monorepo now?